packages/webdriver/webdriver.d.ts

```diff
@@ -397,6 +400,12 @@ declare namespace WebDriver {
         'appium:printPageSourceOnFindFailure'?: boolean;
     }
 
+    // Browserstack General Capabilities
+    export interface BrowserstackCapabilities {
+        os_version?: string;
+        osVersion?: string;
+    }
+
```

Comment:
This has been added in a recent PR: https://github.com/webdriverio/webdriverio/blob/master/packages/webdriver/webdriver.d.ts#L567-L589
```suggestion
```